juego.py

```diff
@@ -15,7 +14,9 @@
 from gi.repository import Gtk
 import os
 import gi
+import sys
```

Comment:
`sys` should probably be the first import

Reply:
oh I forgot to remove, it is not needed in this file.